aten/src/ATen/native/cudnn/Conv.cpp

```diff
@@ -657,7 +657,13 @@ class AlgoIterator {
     if (args.params.dataType == CUDNN_DATA_HALF) {
       perfResults[0].mathType = CUDNN_TENSOR_OP_MATH;
     } else {
+#if defined(CUDNN_VERSION) && CUDNN_VERSION >= 8000
+      if (!at::globalContext().useTF32CuDNN()) {
+        perfResults[0].mathType = CUDNN_FMA_MATH;
+      }
+#else
       perfResults[0].mathType = CUDNN_DEFAULT_MATH;
+#endif
```

Comment:
The above set of conditionals won't touch `mathType` at all `#if defined(CUDNN_VERSION) && CUDNN_VERSION >= 8000` and `at::globalContext().useTF32CuDNN()` is true.  If `mathType` is `CUDNN_DEFAULT_MATH` by default (which would make sense to me...) maybe that's fine, but maybe we want to ensure it's explicitly set for all possible compile and runtime options, ie
```
#if defined(CUDNN_VERSION) && CUDNN_VERSION >= 8000
      if (!at::globalContext().useTF32CuDNN()) {
        perfResults[0].mathType = CUDNN_FMA_MATH;
      } else
#else
      {
        perfResults[0].mathType = CUDNN_DEFAULT_MATH;
      }
#endif
```

Reply:
I believe this is resolved